smithery.yaml

```
@@ -0,0 +1,17 @@
+    # JSON Schema defining the configuration options for the MCP.
+    type: object
+    required:
+      - githubPersonalAccessToken
+    properties:
+      githubPersonalAccessToken:
+        type: string
+        description: GitHub Personal Access Token for authentication.
+  commandFunction:
+    # A function that produces the CLI command to start the MCP on stdio.
+    |-
+    (config) => ({command:'node', args:['dist/index.js'], env:{GITHUB_PERSONAL_ACCESS_TOKEN: config.githubPersonalAccessToken}})
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)